tests/procmacro.rs

```diff
@@ -0,0 +1,17 @@
+use oracle::RowValue;
+
+#[test]
+fn procmacro_ok_shadow() {
+    #[allow(dead_code)]
+
+    enum AnotherResult<T, E> {
```

Comment:
Could you remove the empty line between `#[allow(dead_code)]` and `enum AnotherResult<T, E>`?

The attribute `#[allow(dead_code)]` qualifies `enum AnotherResult`. So the attribute should be followed by the enum without blank lines.

---
What I intended [here](https://github.com/kubo/rust-oracle/pull/62#issuecomment-1195513698) is:
```rust
#[allow(dead_code)]
fn procmacro_ok_shadow() {
    enum AnotherResult<T, E> {
```
However the following code is also acceptable.
```rust
#[test]
fn procmacro_ok_shadow() {
    #[allow(dead_code)]
    enum AnotherResult<T, E> {
```